frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/CreateCluster.tsx

```diff
@@ -325,6 +325,36 @@ export default function CreateCluster(props: { infrastructureType: ClusterInfras
       case 'connection':
         setConnectionControl(control)
         break
+      case 'namespace':
+        control.validation.contextTester = (
+          active: string | undefined,
+          _controlData: object,
+          templateObjectMap: { [x: string]: { HostedCluster: { $raw: { metadata: { namespace: any; name: any } } }[] } }
+        ) => {
+          if (infrastructureType === Provider.kubevirt && templateObjectMap['<<main>>'].HostedCluster[0]) {
```

Comment:
This is common code, so you are correct to check that we are only making this change for `Provider.kubevirt`... but is the check in the correct place? Currently, I don't think any other provider has a `namespace` control, but what would happen if one did?